tests/conftest.py

```diff
@@ -90,6 +90,18 @@ def copy_ireq_dependencies(self, source, dest):
         # No state to update.
         pass
 
+    @property
+    def options(self):
+        """Not used"""
+
+    @property
+    def session(self):
+        """Not used"""
+
+    @property
+    def finder(self):
+        """Not used"""
```

Comment:
I think we might as well type these methods too.

While at first it might seem silly to bother typing tests, it:

1. Helps show we're using the pip-tools API correctly and in a way representative to how it should be used in production.
2. Helps show we're using third party APIs correctly.

I think it is fine to be less strict about types in tests, but when it is easy like this, might as well.

Reply:
Seems reasonable. Addressed in 5f43774eacb849e4483fd2d67f5389af1c1fa0ca.